common-types/index.d.ts

```diff
@@ -207,6 +207,7 @@ interface Shoutout {
   readonly timestamp: number;
   readonly hidden: boolean;
   readonly uuid: string;
+  readonly images?: string[];
```

Comment:
Make sure you update `DBShoutout` in [DataTypes.d.ts](https://github.com/cornell-dti/idol/blob/f64bb993ab2646586e5f476f375368418aab5ce6/backend/src/types/DataTypes.d.ts#L11-L19) as well.